src/wire/mod.rs

```diff
@@ -272,6 +278,12 @@ pub use self::dns::{
     Repr as DnsRepr, Type as DnsQueryType,
 };
 
+#[cfg(feature = "proto-ipsec-ah")]
+pub use self::ipsec_ah::{Packet as IPSecAuthHeaderPacket, Repr as IPSecAuthHeaderRepr};
+
+#[cfg(feature = "proto-ipsec-esp")]
+pub use self::ipsec_esp::{Packet as IPSecEspPacket, Repr as IPSecEspRepr};
```

Comment:
Maybe the letter `P` should be lowercase, then it matches the other types that have IP in their name.